### DISCUSSION TITLE
Single lined binary modelling on PHOEBE

### DISCUSSION THREAD

**Discussion by danangsetyo5364** [phoebe-project/phoebe2#857] [Questions]
Hallo! I am still new in eclipsing binary modelling, and still in learning curve, so I am sorry for my question. Currently I am working with SB1 light and RV curve, I want to find absolute parameters for my object, especially mass ratio. I have run PHOEBE and got some value and good fit on it. But my question is the q or mass-ratio from PHOEBE reliable enough?? Do I need another method to validate it? Thanks in advance!

**Comment by aprsa** [comment]
Hi and welcome to the EB world! :)

No, you cannot trust the mass ratio coming out of an SB1 system. You need an SB2 system for that. In some rare and specific circumstances (totally eclipsing contact binaries) you may get an estimate from photometry, but that estimate will be inevitably less accurate than an SB2 would provide. Some like to perform a q-search to get an estimate for the mass ratio, but that is never appropriate because it does not account for the covariances at all, which is a huge issue.
